UPGRADE.md

```diff
@@ -8,6 +8,15 @@ awareness about deprecated code.
 
 # Upgrade to 5.0
 
+## BC BREAK: Removed support for the `version` column platform option:
```

Comment:
https://github.com/doctrine/dbal/commit/2d95a0f5e11569f7fdbb019b3ecb5203333f629c introduced colons in titles. They look weird to me. Let us drop them?

Reply:
Oh, the trailing ones. Yeah, they look weird in a header.